editor/editor_help.h

```diff
@@ -38,6 +38,7 @@
 #include "scene/gui/margin_container.h"
```

Comment:
This can be removed now.

Reply:
Done. I had to fix a few files that relied on transitive includes.